Splitter/EarlGreyWelcomeScreenTests/EarlGreyWelcomeScreenTests.swift

```diff
@@ -0,0 +1,273 @@
+//
+//  WelcomeScreenTests.swift
+//  WelcomeScreenTests
+//
+//  Created by Wayne Rumble on 14/09/2017.
+//  Copyright © 2017 Wayne Rumble. All rights reserved.
+//
+
+import XCTest
+import EarlGrey
+@testable import Splitter
+
+// swiftlint:disable type_body_length
+class WelcomeScreenTests: XCTestCase {
+    
+    let password = "password"
+    let shortPassword = "passw"
+    let differentPassword = "differentPassword"
+    let wrongPassword = "wrongpassword"
+    let registeredEmail = "alreadyregistereduser@email.com"
+    let unregisteredEmail = "unregistered@email.com"
+    let invalidEmail = "invalid@email"
+    
+    override func setUp() {
+        super.setUp()
+        restartApp()
+    }
+    
+    func testHasLogoLabel() {
+        let passwordTextField = grey_accessibilityID(AccesID.passwordTextField)
```

Comment:
Do these not need updating with `AccessID` (the typo fix you've done to the enum :) )